pkg/api/v1/conversion.go

```diff
@@ -191,6 +191,7 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 				"metadata.uid",
 				"spec.nodeName",
```

Comment:
not this PR :). But when `spec.host` can be removed? If not sure, suggest to depreciated since 1.8;  and remove it in 1.10 (?).

Reply:
I wouldn't expect it to be removed